<launch>
  <arg name="save_dir" default="$(find open_autonomous_tractor)/datasets/recorded_poses" />
  
  <arg name="csv_filename" default="recorded_poses.csv" />
  
  <arg name="frame_id" default="map" />
  
  <node pkg="open_autonomous_tractor" type="waypoint_recorder_gui.py" name="waypoint_recorder_gui" output="screen">
    <param name="save_dir" value="$(arg save_dir)" />
    <param name="csv_filename" value="$(arg csv_filename)" />
    <param name="frame_id" value="$(arg frame_id)" />
  </node>
</launch>